src/docs/sdk/features.mdx

```diff
@@ -396,6 +396,59 @@ The `fingerprints` field should be set to `["$operationName", "$operationType",
 }
 ```
 
+## GraphQL Performance
+
+The GraphQL Performance integration should match the guidelines for [GraphQL Client Integrations](/sdk/features/#graphql-client-integrations) with a few differences:
+
+The transaction's name should be set with the GraphQL operation name, if possible, otherwise fallback to something unique that makes sense, e.g. the canonical name of the actual/generated class.
+
+The transaction's description should be set with the GraphQL operation name, operation type and status code, if possible.
+
+The `request.api_target` should be set with `graphql`.
+
+The `request.data` should be set with the raw GraphQL request payload, if possible.
+
+The `contexts.response.data` should be set with the raw GraphQL response payload, only if there were `errors`.
+
+Spans should be created for [resolvers](https://www.apollographql.com/docs/apollo-server/data/resolvers/), if possible.
+
+Spans should be created for [data loaders](https://graphql.org/learn/best-practices/#server-side-batching-caching/), if possible.
+
+The operation type should follow the [Span Data Conventions](/sdk/performance/span-data-conventions/).
```

Comment:
Do we want to add the [conventions for GraphQL](https://opentelemetry.io/docs/specs/otel/trace/semantic_conventions/instrumentation/graphql/) in this PR?

Reply:
I don't know if our convention matches 1:1 with OTel but I'd at least link to it, yes.
Wanna make a suggestion?

Reply:
I don't understand, do we already have one for GraphQL?

Reply:
I fixed the link, this is only about the operation type.
for extra attributes in the `data` field, makes sense to follow https://opentelemetry.io/docs/specs/otel/trace/semantic_conventions/instrumentation/graphql/ and maybe add it to the span data conventions.

Reply:
Added a new line for this.